geopandas/tools/_show_versions.py

```diff
@@ -22,6 +22,53 @@ def _get_sys_info():
     return dict(blob)
 
 
+def _get_C_info():
+    """Information on system PROJ, GDAL, GEOS
+    Returns
+    -------
+    c_info: dict
+        system PROJ information
+    """
+    try:
+        import pyproj
+        from pyproj.exceptions import DataDirError
+        proj = pyproj.proj_version_str
+        try:
+            proj_dir = pyproj.datadir.get_data_dir()
+        except DataDirError:
+            proj_dir = None
+    except ImportError:
+        proj = None
+        proj_dir = None
+
+    try:
+        import shapely._buildcfg
+        geos = '{}.{}.{}'.format(*shapely._buildcfg.geos_version)
+        geos_dir = shapely._buildcfg.geos_library_path
+    except ImportError:
+        geos = None
+        geos_dir = None
+
+    try:
+        import fiona
+        gdal = fiona.env.get_gdal_release_name()
+        gdal_dir = fiona.env.GDALDataFinder().search()
+    except ImportError:
+        gdal = None
+        gdal_dir = None
+
+    blob = [
+            ("GEOS", geos),
+            ("GEOS lib", geos_dir),
+            ("GDAL", gdal),
+            ("GDAL dir", gdal_dir),
+            ("PROJ", proj),
+            ("PROJ dir", proj_dir)
```

Comment:
```suggestion
            ("PROJ data dir", proj_dir)
```

to be more explicit what kind of dir it is?